starfyre/__main__.py

```diff
@@ -71,7 +72,7 @@ def main(path, build):
             [sys.executable, "-m", "build"],
             cwd=path,
             stdout=subprocess.PIPE,
-            stderr=subprocess.PIPE,
+            stderr=None,
```

Comment:
I added this line because the process was not sending any error output to the console. I researched more and found that setting the `stderr` to `None` will ensure we show error output to the users of `Starfyre`

Reply:
Niceee